client/style.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const Title = styled.h1`
+  display: flex;
```

Comment:
You might not need this property here. I believe only the parent container need display: flex. But I might be wrong. I look at [https://css-tricks.com/snippets/css/a-guide-to-flexbox/](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) when I need help with flex.